css/style.css

```diff
@@ -0,0 +1,43 @@
+Body{
+    background-color: grey;
+}
+
+#title{
+      text-align: center;
+      font-size: 24px;
+}
+#img-div{
+    text-align: center;
+    display: block;
+    filter: grayscale(100%);
+    width: 100%;
+    background-color: white;
+    padding: 20px;
+    margin: 15px;  
+ }
+ 
+ img{
+   padding-top: 10px;
+   max-width: 100%; 
+ }
```

Comment:
- [ ] [OPTIONAL] Good job 💎, you used [Github inline comments feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) as previous reviewers said. If you want to make you your review better I suggest using some emojis (around 1-3) which makes your commits human friendly👽🤖